From 73094608b8e214952444fb104651704c98a37aeb Mon Sep 17 00:00:00 2001
From: Christoffer Nielsen <cn@obviux.dk>
Date: Fri, 19 Jun 2020 13:48:22 +0200
Subject: [PATCH] ALSA: usb-audio: Add registration quirk for Kingston HyperX Cloud Flight S
Git-commit: 73094608b8e214952444fb104651704c98a37aeb
Patch-mainline: v5.8-rc3
References: git-fixes

Similar to the Kingston HyperX AMP, the Kingston HyperX Cloud
Alpha S (0951:0x16ea) uses two interfaces, but only the second
interface contains the capture stream. This patch delays the
registration until the second interface appears.

Signed-off-by: Christoffer Nielsen <cn@obviux.dk>
Cc: <stable@vger.kernel.org>
Link: https://lore.kernel.org/r/CAOtG2YHOM3zy+ed9KS-J4HkZo_QGzcUG9MigSp4e4_-13r6B=Q@mail.gmail.com
Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index c495e720e2f1..54d4b4b5bd11 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1857,6 +1857,7 @@ struct registration_quirk {
 static const struct registration_quirk registration_quirks[] = {
 	REG_QUIRK_ENTRY(0x0951, 0x16d8, 2),	/* Kingston HyperX AMP */
 	REG_QUIRK_ENTRY(0x0951, 0x16ed, 2),	/* Kingston HyperX Cloud Alpha S */
+	REG_QUIRK_ENTRY(0x0951, 0x16ea, 2),	/* Kingston HyperX Cloud Flight S */
 	{ 0 }					/* terminator */
 };
 
-- 
2.16.4

